spacy/cli/templates/quickstart_training.jinja

```diff
@@ -3,7 +3,7 @@ the docs and the init config command. It encodes various best practices and
 can help generate the best possible configuration, given a user's requirements. #}
 {%- set use_transformer = hardware != "cpu" and transformer_data -%}
 {%- set transformer = transformer_data[optimize] if use_transformer else {} -%}
-{%- set listener_components = ["tagger", "morphologizer", "parser", "ner", "textcat", "textcat_multilabel", "entity_linker", "span_finder", "spancat", "spancat_singlelabel", "trainable_lemmatizer"] -%}
+{%- set listener_components = ["tagger", "morphologizer", "parser", "ner", "textcat", "textcat_multilabel", "entity_linker", "span_finder", "spancat", "spancat_singlelabel", "trainable_lemmatizer", "llm"] -%}
```

Comment:
`llm` shouldn't be in this list of components with listeners, which will cause `tok2vec` or `transformers` to be added (depending on the rest of the template, of course).

Reply:
Makes sense. I vaguely remember that the LLM block was not output properly if "llm" was not in `listener_components`. I'll check.